static/coffee/controllers.coffee

```diff
@@ -573,12 +588,12 @@ controllers.controller('CaseController', ['$scope', '$window', '$timeout', 'Case
   #----------------------------------------------------------------------------
 
   $scope.onWatch = () ->
-    UtilsService.confirmModal("Receive notifications for activity in this case?").then(() ->
+    UtilsService.confirmModal("Receive alerts for activity in this case?").then(() ->
```

Comment:
I just took a peek and there's only ~15 calls I think - but some things you'll need to add/check:
- a danger param which sets the styling of the OK button to danger
- that prompt strings can include markup, e.g. `"Apply the label <strong>" + label.name + "</strong> to the selected messages?"` - or there's at least an easy way to do that without creating an entire modal template